cmd/kubeadm/app/master/apiclient.go

```diff
@@ -56,17 +58,35 @@ func CreateClientAndWaitForAPI(file string) (*clientset.Clientset, error) {
 			return false, nil
 		}
 		n := &nodeList.Items[0]
-		if !v1.IsNodeReady(n) {
-			fmt.Println("[apiclient] First node has registered, but is not ready yet")
-			return false, nil
+		if v1.IsNodeReady(n) {
+			firstNodeIsReady = true
+		} else {
```

Comment:
Pushed a cleaned up version which I'm testing now, although we may not end up needing this PR...